conf/defaults.ini

```diff
@@ -843,7 +843,14 @@ skip_org_role_sync = false
 sync_cron = "0 1 * * *"
 active_sync_enabled = true
 
-#################################### AWS ###########################
+#################################### Auth SSO Settings ###########################
+[auth.sso_settings]
+# interval for reloading the SSO Settings from the database
+# useful in high availability setups running multiple Grafana instances
+# set to 0 to disable this feature
+reload_interval = 1m
```

Comment:
Could you please rename it to `[sso_settings]`? Currently the `auth.` prefix is only used by the different providers and this could be confusing.

Reply:
Done.